tests/cli-tests/determinism/multi.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+. "$COMMON/platform.sh"
+
+set -e
+
+datagen -g0 > file0
+datagen -g1 > file1
+datagen -g1000 > file1000
+datagen -g100000 > file100000
+
+for level in $(seq 1 19); do
+    echo "$level"
+    zstd -f -$level file0 file1 file1000 file100000
```

Comment:
Why is the same test repeated 4 times ?

Reply:
Different ordering of the files

Reply:
OK, so this is checking the absence of impact on the compressed output when compressing multiple payloads using the same context.

Reply:
Yeah, I'll add a comment here. It is a very basic test.